documentation/frontend/logging.md

```diff
@@ -0,0 +1,56 @@
+# Next JS Logging
+
+## Background
+
+The Next JS application logs to std out where messages are picked up by the Fluent Bit sidecar and pushed to both Cloudwatch and New Relic.
```

Comment:
Does the sidecar push to Cloudwatch or just to New Relic and the stdout is automatically caught by Cloudwatch?

Reply:
not sure why, but my understanding is that the std out doesn't make it to cloudwatch without fluent bit specifically pushing it there. @coilysiren would know more

Reply:
stdout gets pushed to sidecar, sidecar pushes it to cloudwatch and new relic

Reply:
so yeah, what Doug said